webview/src/setup/components/studio/Settings.tsx

```diff
@@ -19,7 +19,7 @@ export const Settings: React.FC<
   return (
     <EmptyState isFullScreen={false}>
       <div>
-        <h1>Studio Settings</h1>
+        <h1>DVC Studio Settings</h1>
```

Comment:
I updated title-like text but I'm not sure if we should be referring to DVC Studio everywhere. Other places:

* in Command Pallete commands
* link description in experiments table
* toasts
* description tooltip in setup

Reply:
can you please share screenshots to review those places?

Reply:
Sure! Here are other places where we mention "Studio":  

<img width="487" alt="image" src="https://github.com/iterative/vscode-dvc/assets/43496356/849969ae-e814-4867-bed9-5ad1b941dca2">

<img width="510" alt="image" src="https://github.com/iterative/vscode-dvc/assets/43496356/64be65d3-f467-416c-b194-9ec37566e6bc">

<img width="634" alt="image" src="https://github.com/iterative/vscode-dvc/assets/43496356/a29fe777-0211-4495-8605-9125deb42483">

<img width="359" alt="image" src="https://github.com/iterative/vscode-dvc/assets/43496356/5be938c2-918a-4ff7-bfa5-2e2e3920f87a">

<img width="417" alt="image" src="https://github.com/iterative/vscode-dvc/assets/43496356/0b1fe93d-017c-4a70-9ee4-b89fd6a7444b">

<img width="772" alt="image" src="https://github.com/iterative/vscode-dvc/assets/43496356/52b54242-22ed-42e1-bda5-1ce2bf78a74f">

Reply:
I don't have a strong opinion on this tbh. I think it's clear from the context already. @jendefig wdyt? :)